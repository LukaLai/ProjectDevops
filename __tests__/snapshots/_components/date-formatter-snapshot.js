import { render } from '@testing-library/react'
import DateFormatter from '../../../src/app/_components/date-formatter'

it('renders homepage unchanged', () => {
    const { container } = render(<DateFormatter dateString={'2024-04-17T05:35:07.322Z'}/>)
    expect(container).toMatchSnapshot()
})