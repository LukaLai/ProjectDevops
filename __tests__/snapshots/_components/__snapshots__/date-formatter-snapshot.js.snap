// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <time
    datetime="2024-04-17T05:35:07.322Z"
  >
    April	17, 2024
  </time>
</div>
`;
